bindings/Gtk4LayerShell/pkg.info

```diff
@@ -0,0 +1,22 @@
+{
+    "name": "gi-gtk4-layer-shell",
+    "version": "0.1.0",
+    "description": "Bindings for gtk4-layer-shell, autogenerated by haskell-gi.",
+    "synopsis": "gtk4-layer-shell bindings",
+    "author": null,
+    "girName": "Gtk4LayerShell",
+    "girVersion": "1.0",
+    "girOverrides": "Gtk4LayerShell.overrides",
+    "baseVersion": "base >= 4.9 && <5",
+    "distributionPackages": {
+        "fedora": [],
+        "ubuntu": [
+            "libgtk4-layer-shell-dev"
```

Comment:
If the package does not exist in Ubuntu 22.04 (which is the version currently used for the CI) you can add a line `"ubuntu-ci": ["skip"],` which should fix the CI. (If this does not work right away don't bother too much with it, I'll fix it after merging.)